test/python/app_seat_service_client/app/AppManifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "manifestVersion": "v3",
+    "name": "SampleApp",
+    "interfaces": [
+        {
+            "type": "grpc-interface",
+            "config": {
+                "src": "https://raw.githubusercontent.com/eclipse/kuksa.val.services/main/seat_service/proto/sdv/edge/comfort/seats/v1/seats.proto",
```

Comment:
please use the new eclipse-kuksa organisation

Reply:
Done